gino/declarative.py

```diff
@@ -134,10 +134,18 @@ def select(cls, *args):
 
     @classmethod
     async def get_or_404(cls, id_, bind=None, timeout=None):
+        try:
+            from sanic.exceptions import NotFound
+        except ModuleNotFoundError:
+            from .exceptions import SanicNotInstallError
+            raise SanicNotInstallError(
+                '''Sanic has not been installed yet. Please run "pip install 
+                sanic" to solve this error.'''
```

Comment:
- I think a simple explanation of Sanic is not installed is enough. The way how to install it is not necessary.
- The triple-quote string should not be used like this. It's for long paragraph, not a short sentence, and the line-breaks, and indents will be included in the output. I don't think that's what you want.